SwanProjects/swanprojects/utils.py

```diff
@@ -0,0 +1,107 @@
+import json
+import os
+
+
```

Comment:
Some of these functions (like is project, is this a project path, etc) exist in swancontens. I believe we should re-use them.

Reply:
implemented with contents manager.